Documentation/ceph-upgrade.md

```diff
@@ -479,62 +329,66 @@ updated we wait for things to settle (monitors to be in a quorum, PGs to be clea
 MDSs, etc.), then only when the condition is met we move to the next daemon. We repeat this process
 until all the daemons have been updated.
 
-## Ceph images
+### Ceph images
 
 Official Ceph container images can be found on [Docker Hub](https://hub.docker.com/r/ceph/ceph/tags/).
 These images are tagged in a few ways:
 
-* The most explicit form of tags are full-ceph-version-and-build tags (e.g., `v13.2.6-20190830`).
+* The most explicit form of tags are full-ceph-version-and-build tags (e.g., `v14.2.5-20191210`).
   These tags are recommended for production clusters, as there is no possibility for the cluster to
   be heterogeneous with respect to the version of Ceph running in containers.
-* Ceph major version tags (e.g., `v13`) are useful for development and test clusters so that the
+* Ceph major version tags (e.g., `v14`) are useful for development and test clusters so that the
   latest version of Ceph is always available.
 
 **Ceph containers other than the official images from the registry above will not be supported.**
 
-## Example upgrade to Ceph Nautilus
+### Example upgrade to Ceph Nautilus
 
-### 1. Update the main Ceph daemons
+#### 1. Update the main Ceph daemons
 
 The majority of the upgrade will be handled by the Rook operator. Begin the upgrade by changing the
 Ceph image field in the cluster CRD (`spec:cephVersion:image`).
 
-```console
-NEW_CEPH_IMAGE='ceph/ceph:v14.2.2-20190830'
+```sh
```

Comment:
Is there a difference between `sh` and `console` rendering?

Reply:
`sh` seems to highlight the variable, whereas `console` does not unless it is prefixed with `#` or `$`.